solvebio/global_search.py

```diff
@@ -193,6 +193,6 @@ def subjects_count(self):
         """If entity seaarch is performed returns the list of subjects"""
```

Comment:
Docstring is out of date

Reply:
Nice, thanks, missed that one. I'll double-check in the r client as well